kythe/rust/extractor/src/lib.rs

```diff
@@ -29,75 +36,81 @@ use std::path::PathBuf;
 /// first element must be an empty string.
```

Comment:
Simply stating this doesn't feel like enough explanation for what seems like a strange arbitrary constraint. What's the "why" for this?

Reply:
The first argument always gets ignored, which is why it should be an empty string. But I agree, this comment should be better.